pkgs/development/python-modules/vdirsyncer/default.nix

```diff
@@ -53,11 +32,13 @@ buildPythonPackage rec {
     aiohttp-oauthlib
   ];
 
-  nativeBuildInputs = [
-    setuptools-scm
+  nativeBuildInputs = with python3Packages; [
+    pythonRelaxDepsHook
   ];
 
-  nativeCheckInputs = [
+  pythonRelaxDeps = true;
```

Comment:
Be specific on the exact packages to relax, e.g. `pythonRelaxDeps = [ "foo" ];`

Reply:
I had another look at upstream and it seems all that is unnecessary now. I removed the code.